src/CMakeLists.txt

```diff
@@ -117,10 +132,12 @@ if(HAVE_INTEL)
     set(object_format "elf64")
   endif()
   set(CMAKE_ASM_FLAGS "-f ${object_format}")
-  include(CheckYasm)
-  check_yasm_support(${object_format}
-    HAVE_GOOD_YASM_ELF64
-    HAVE_BETTER_YASM_ELF64)
+  if(NOT WIN32)
+    include(CheckYasm)
```

Comment:
why? if yasm is not find, i think it's fine if neither HAVE_GOOD_YASM_ELF64 nor HAVE_BETTER_YASM_ELF64 is defined. the message does not hurt. it's just informative.

Reply:
It's one of those cases in which the native libraries or tools are located but cannot be used when cross compiling. I'll add an inline comment explaining this.

Reply:
@petrutlucian94 do you think it might be a good idea to disable yasml if we are crosscompiling?

Reply:
Hmm, not entirely sure about BSD.